easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-4.0.1.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+
+name = 'vsc-mympirun'
+version = '4.0.1'
+
+homepage = 'https://github.com/hpcugent/vsc-mympirun'
+description = """VSC-tools is a set of Python libraries and scripts that are commonly used within HPC-UGent."""
+
+# we build this to work with every python version
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['b5b08e6291a68802d29c344d5807e4c2a1945375055b57c819721151775789f0']
+
+builddependencies = [('vsc-install', '0.10.26')]
+dependencies = [('vsc-base', '2.5.8')]
+
+osdependencies = ['python-setuptools']
+
+# we ship something in bin/fake
+modextrapaths = {'PATH': 'bin/fake'}
+
+# don't check for import of vsc.mympirun, since that may fail if vsc-base is picked up from the OS
+options = {'modulename': 'vsc'}
+
+pyshortver = '.'.join(SYS_PYTHON_VERSION.split('.')[:2])
```

Comment:
Only when `Python` is listed as a dependency, not when you're using the system Python like we're doing here...